images/rook/Dockerfile

```diff
@@ -16,5 +16,11 @@ FROM BASEIMAGE
 
 ADD rook rookflex /usr/local/bin/
 
+RUN mkdir -p /var/lib/rook && \
+    chmod -R g=u /var/lib/rook /var/lib/ceph /etc/ceph
```

Comment:
`/var/lib/ceph /etc/ceph` and `/etc/ceph` don't exist since we remove them after the package install so this command will fail. Also isn't this dup of https://github.com/rook/rook/pull/1346/commits/2df0629c8a70a7510a774413e132888d6a1fd627

Reply:
where are we removing `/var/lib/ceph`. Yes its a dupe, I'm experimenting with your patch in this fork which uses deb packages, vs. #1346 which still builds ceph from source.

Reply:
Well in this commit 63fc39bc30ec63cd8b1252cc9dd45f19da0822fe you had something like "remove ceph directories to avoid confusion" but I guess it's gone now. I guess I had an old version.